x-pack/plugins/cases/public/components/create/custom_fields.tsx

```diff
@@ -43,18 +43,20 @@ const CustomFieldsComponent: React.FC<Props> = ({ isLoading, customFieldsConfigu
     }
   );
 
+  if (!customFieldsConfiguration.length) {
+    return null;
+  }
+
   return (
     <EuiFlexGroup direction="column" gutterSize="s">
-      {customFieldsComponents.length ? (
-        <EuiText
-          size="m"
-          css={css`
-            font-weight: ${euiTheme.font.weight.bold};
-          `}
-        >
-          {i18n.ADDITIONAL_FIELDS}
-        </EuiText>
-      ) : null}
+      <h3
```

Comment:
Sorry, what I meant was

```
<EuiText size="m">
    <h3>{i18n.ADDITIONAL_FIELDS}</h3>
</EuiText>
```

Reply:
By `size` I mean the size of the `EuiText`.

Reply:
oh okay 😄